### DISCUSSION TITLE
Missing Circuit 10

### DISCUSSION THREAD

**Discussion by bendinwire** [tagyoureit/nodejs-poolController#1015] [Q&A]
Hi

I have most everything set up in the pool control dashboard, but it is missing one setting - the Path Lights. Autelis and ScreenLogic both can see (and control) this. It is circuit 10. However when I add Circuit 10 (it wasn't auto-detected on setup) and try to control it I get nothing.

See attached pics. Any idea on how I can get it to recognize that circuit?

Thanks in advance
<img width="641" alt="autelis_circuit_details" src="https://github.com/user-attachments/assets/13ee213a-3021-4b3d-8ccb-46f491d9c679">
<img width="666" alt="pool_control_dashboard-8TBH6gI2-G" src="https://github.com/user-attachments/assets/c05166f1-baf3-4ba2-8a94-0125974af9ab">
<img width="909" alt="pool_spa_control_dashboard" src="https://github.com/user-attachments/assets/fcddc58a-4f80-4bc4-a4d8-1d8d9bf9f7a9">

**Comment by rstrouse** [comment]
Please upload a replay so that I can see what has been detected from the IntelliTouch personality kit.
